packit_service/worker/jobs.py

```diff
@@ -49,6 +49,16 @@ def add_to_mapping(kls: Type["JobHandler"]):
     return kls
 
 
+def do_we_process_fedmsg_topic(topic: str) -> bool:
+    """ do we process selected fedmsg topic? """
+    global JOB_NAME_HANDLER_MAPPING
+    for kls in JOB_NAME_HANDLER_MAPPING.values():
+        t = getattr(kls, "topic", "this-is-not-a-real-topic")
+        if t == topic:
```

Comment:
Just wondering if it's more readable for you with the intermediary `t`
(than just `if getattr(kls, "topic", "not-a-real-topic") == topic:`

Reply:
but then we need to catch the exc; ok, let me change